prime-router/src/testIntegration/kotlin/datatests/TranslationTests.kt

```diff
@@ -431,19 +428,10 @@ class TranslationTests {
             }
 
             if (!config.conditionFiler.isNullOrBlank()) {
-                if (config.conditionFiler == "pruneUnwanted") {
-                    val provenance = fhirBundle.entry.first {
-                        it.resource.resourceType.name == "Provenance"
-                    }.resource as Provenance
-                    // pick the only Endpoint reference in sample input
-                    val endpoint = provenance.target.map { it.resource }.filterIsInstance<Endpoint>()[0]
-                    fhirBundle = FHIRTranslator().pruneBundleForReceiver(fhirBundle, endpoint)
-                } else {
-                    fhirBundle = fhirBundle.filterObservations(
-                        listOf(config.conditionFiler),
-                        emptyMap<String, String>().toMutableMap()
-                    )
-                }
+                fhirBundle = fhirBundle.filterObservations(
+                    listOf(config.conditionFiler),
+                    emptyMap<String, String>().toMutableMap()
```

Comment:
I know this came from existing code but is there a reason this is mutable?

Reply:
The function requires a mutable map as a param.